src/components/Card/Card.style.js

```diff
@@ -41,16 +41,23 @@ export const CardWrapper = styled.div`
         border-top-right-radius: 0.5rem;
         border-top-left-radius: 0.5rem;
 
-        .gatsby-image-wrapper, .old-gatsby-image-wrapper{
+        .gatsby-image-wrapper{
+            height:80%;
+            width:70%;
+            margin-top: 0.5rem;
+            margin-left: 3rem;
```

Comment:
Here you have explicitly added margin-left as 3rem which is causing alignment issues on some screen size between desktop and mobile(image is not in center of the card).

Maybe you can try 
`margin:auto;
margin-top:0.5rem; `
<img width="418" alt="image" src="https://github.com/layer5io/layer5/assets/76479104/37578d27-0eb1-4ceb-abcb-1a4c1c24c1af">

Reply:
Well good work @VamshiReddy02 it is working for the desktop screen.